src/pages/SubmissionConfig.js

```diff
@@ -427,7 +444,8 @@ export const SubmissionConfig = ({ methodId }) => {
         submissionId: runSetObject.run_set_id
       })
     } catch (error) {
-      notify('error', 'Error submitting workflow', { detail: await (error instanceof Response ? error.text() : error) })
```

Comment:
TOL: I wonder if it would still be good to show the error in the toast form behind the modal so that the user can still refer to the error after closing the modal and while fixing the error (if they can). What do you think?

Reply:
For posterity, discussed in person and decided to leave out error for now, with potential for revisiting on user feedback